pkg/dev/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM linuxkit/sshd:e108d208adf692c8a0954f602743e0eec445364e
+
+ENV DOCKER_BUCKET get.docker.com
+ENV DOCKER_VERSION 17.05.0-ce
+ENV DOCKER_SHA256 340e0b5a009ba70e1b644136b94d13824db0aeb52e09071410f35a95d94316d9
+
+RUN \
+  apk update && apk upgrade && \
+  apk add --no-cache \
+  curl \
+  && true
+
+RUN set -x \
+	&& curl -fSL "https://${DOCKER_BUCKET}/builds/Linux/x86_64/docker-${DOCKER_VERSION}.tgz" -o docker.tgz \
+	&& echo "${DOCKER_SHA256} *docker.tgz" | sha256sum -c - \
+	&& tar -xzvf docker.tgz \
+	&& mv docker/* /usr/bin/ \
```

Comment:
This includes too much - you just want the `docker` client for dev, not the daemon etc.